src/layouts/docs/docs-layout.module.css

```diff
@@ -83,6 +47,30 @@ heading with only a couple words, the element can become too narrow.
     }
   }
 
+  & h1,
+  & h2,
+  & h3,
+  & h4,
+  & h5,
+  & h6 {
+    /*
+    These ::before elements account for the height of the top sticky bars
+    when following #anchor-links to headings. Without the offset caused
+    by these elements, we'd jump to headings that would be hidden underneath
+    the top sticky bars.
+    */
+    & a:global(.__target-h)::before,
+    &::before {
+      display: block;
+      content: '\200B';
+      height: calc(var(--navigation-header-height) + var(--alert-bar-height));
+      margin-top: calc(
+        -1 * calc(var(--navigation-header-height) + var(--alert-bar-height))
+      );
+      visibility: hidden;
+    }
+  }
```

Comment:
This is really neat! Nicely done ✨

Reply:
for consideration: I think something similar to this is how we're handling it on the current docs sites: https://github.com/hashicorp/nomad/blob/main/website/pages/style.css#L33-L49